lumicks/pylake/detail/image.py

```diff
@@ -3,6 +3,60 @@
 import numpy as np
 
 
+class ImageMetaData:
+    """Image metadata
+
+    Parameters
+    ----------
+    pixel_size_x : float
+        Horizontal pixel size [nm]
+    pixel_size_y : float
+        Vertical pixel size [nm]
+        When omitted, assumed to be identical to pixel_size_x
+    pixel_time : float
+        How long does pixel acquisition take [ms]
+    """""
+    def __init__(self, pixel_size_x=1.0, pixel_size_y=None, pixel_time=1.0):
+        self._pixel_time = pixel_time
+        self._pixel_size_x = pixel_size_x
+        if pixel_size_y:
+            self._pixel_size_y = pixel_size_y
+        else:
+            self._pixel_size_y = pixel_size_x
+
+    @staticmethod
+    def from_dataset(json=None):
+        """
+        Fetch metadata from json structure
+
+        Parameters
+        ----------
+        json : json structure containing kymograph metadata
```

Comment:
Type goes after the colon. The description goes on the next line.

Reply:
Modified.